src/Grassmann.jl

```diff
@@ -0,0 +1,263 @@
+using LinearAlgebra: svd, qr, Diagonal, det
+import LinearAlgebra: norm
+@doc doc"""
+    Grassmann{n,k} <: Manifold
+
+The Grassmann manifold $\operatorname{Gr}(n,k)$ consists of all $k$-dimensional
+subspaces of $\mathbb F^n$, where $\mathbb F \in \{\mathbb R, \mathbb C\}$ is
+either the real- (or complex-) valued subspaced of the $n$-dimensional (complex)
+space.
```

Comment:
This description is slightly unclear. $\mathbb{C}^n$ is $2n$-dimensional, right? Could you reflect this in the description? Also, `subspaced` should be replaced with `subspaces`.

Reply:
I rephrased this.